services/core-web/webpack.config.ts

```diff
@@ -35,7 +35,8 @@ const PATHS = {
 };
 
 const BUILD_FILE_NAMES = {
-  css: "style/[name].[contenthash:4].css",
+  css: "style/[name].[contenthash:8].css",
+  cssChunkName: "style/[name].[chunkhash:8].css",
```

Comment:
I remembered we updated this in Minespace a while ago, but looks like we didn't in Core!

Do not know why, but it now works. Thanks anonymous stackoverflow post

https://stackoverflow.com/questions/71142869/invalid-js-in-webpack-5-bundle-some-numbers-from-source-code-are-replaced-by-nu

Reply:
nice catch :)

Reply:
Oh!  Nice!